src/languages/es-ES.ts

```diff
@@ -887,6 +887,7 @@ export default class extends Language {
 			NONE: 'Ninguno'
 		},
 		COMMAND_FFXIV_NO_CHARACTER_FOUND: 'Lo siento, pero no pude encontrar un personaje con ese nombre.',
+		COMMAND_FFXIV_INVALID_SERVER: 'Lo siento, pero el nombre del servidor no es válido. Por favor verifica que está bien escrito con las mayúsculas correctas.',
 		COMMAND_FFXIV_NO_ITEM_FOUND: 'Lo siento, pero no pude encontrar un elemento con esa consulta.',
```

Comment:
```suggestion
		COMMAND_FFXIV_NO_CHARACTER_FOUND: `${REDCROSS} Lo siento, pero no pude encontrar un personaje con ese nombre.`,
		COMMAND_FFXIV_INVALID_SERVER: `${REDCROSS} Lo siento, pero el nombre del servidor no es válido. Por favor verifica que está bien escrito con las mayúsculas correctas.`,
		COMMAND_FFXIV_NO_ITEM_FOUND: `${REDCROSS} Lo siento, pero no pude encontrar un elemento con esa consulta.`,
```

If this code is being touched anyway, might as well partially cover an open issue